pkg/config/teeconfig/teeconfig.go

```diff
@@ -158,11 +158,6 @@ func (t *teeConfig) GetFloat64(key string) float64 {
 	return t.baseline.GetFloat64(key)
 }
 
-// GetTime wraps Viper for concurrent access
-func (t *teeConfig) GetTime(key string) time.Time {
-	return t.baseline.GetTime(key)
-}
-
 // GetDuration wraps Viper for concurrent access
```

Comment:
Not needed in this PR, but we should probably update these comments at some point.